daml/Marketplace/Clearing/Market/Model.daml

```diff
@@ -0,0 +1,51 @@
+module Marketplace.Clearing.Market.Model where
+
+import DA.Finance.Types
+import DA.Next.Set (Set)
+
+template FairValueCalculationRequest
+  with
+    operator : Party
+    provider : Party
+    customer : Party
+    optListingIds : Optional [Text]
```

Comment:
Probably worth `newtype`ing this. How are we currently identifying listings?

Reply:
Elsewhere they've been `listingId: Text`, but I think using `Id` might be better so you can `key` the `Listing` with the `Id`. Currently if you're trying to do a `fetchByKey` of a listing, you wouldn't know who the provider/customer is to do the lookup.

Reply:
I am addressing this is a separate PR